src/go/rdctl/cmd/shell.go

```diff
@@ -116,21 +127,37 @@ func setupLimaHome() error {
 const restartDirective = "Either run 'rdctl start' or start the Rancher Desktop application first"
 
 func checkLimaIsRunning(commandName string) bool {
-	// Ignore error messages; none are expected here
-	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}")
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Start()
+	if err == nil {
+		err = cmd.Wait()
+	}
```

Comment:
Why not use

```suggestion
	err := cmd.Run()
```

Reply:
Because of how I read up on capturing `stdout` and `stderr` from a process in Go. Clearly the docs were wrong.